README.md

```diff
@@ -2,6 +2,18 @@
 The Juvix Emacs mode
 
 ## Installation
+
+### Using straight.el
+
+This installs juvix-mode, to your emacs config
+
+```emacs-lisp
+(use-package juvix-mode :straight (:host github :repo "anoma/juvix-mode")
+  :mode ("\\.\\(juvix\\)$" . juvix-mode))
```

Comment:
```
  :mode ("\\.\\(juvix\\)$" . juvix-mode))
```
Just to confirm. This line is necessary even though we have the following in `juvix-mode.el`?
```
(add-to-list 'auto-mode-alist '("\\.juvix\\'" . juvix-mode))
(add-to-list 'auto-mode-alist '("\\.juvix.md\\'" . juvix-mode))
```

If so, we'd need to add the `.juvix.md` extension as well.

Reply:
Yeah, for some reason it doesn't work for me without it

Reply:
Probably a bug in how it's registered in the elisp

Reply:
then could you add the `juvix.md` extension to trigger the mode as well?